dapp/src/client/scripts/app.jsx

```diff
@@ -125,17 +125,13 @@ export class App extends React.Component {
   getAvailableContracts() {
     this.getContracts("/contract/available_contracts")
       .then(data => {
-        const contracts = {}
```

Comment:
We no longer store keys: contracts here, but merely an array of contracts instead

Reply:
This breaks update...